lib/active_record_doctor/detectors/missing_unique_indexes.rb

```diff
@@ -59,13 +85,19 @@ def supported_validator?(validator)
           validator.options.fetch(:case_sensitive, true)
       end
 
-      def unique_index?(table_name, columns, scope)
+      def unique_index?(table_name, columns, scope = nil)
         columns = (Array(scope) + columns).map(&:to_s)
 
         indexes(table_name).any? do |index|
           Array(index.columns).to_set == columns.to_set && index.unique
         end
       end
+
+      def ignore_columns
+        @ignore_columns ||= config(:ignore_columns).map do |column|
+          column.gsub(" ", "")
+        end
+      end
```

Comment:
Why do we need this?

Reply:
That was moved from `detect` method. I think, it is needed to be able to work with `User(organization_id, email, role)` and `User(organization_id,email,role)` (no spaces) the same way.